src/types/button.ts

```diff
@@ -11,3 +15,14 @@ export enum ButtonSize {
   LARGE = 'large',
   MINI = 'mini',
 }
+
+export interface ButtonProps {
+  disabled?: boolean;
+  icon?: React.ReactElement;
+  loading?: boolean;
+  kind?: ButtonKind;
+  onPress?: (event: GestureResponderEvent) => void;
+  size?: ButtonSize;
+  style?: ViewStyle;
```

Comment:
This is unexpected. The button should render appropriate experience based on kind.